skyportal/tests/models/test_permissions.py

```diff
@@ -0,0 +1,3207 @@
+    accessible = problematic_assignment.is_accessible_by(user, mode="delete")
+    assert accessible
+
+
+def test_user_group2_create_problematic_assignment(user_group2, problematic_assignment):
+    accessible = problematic_assignment.is_accessible_by(user_group2, mode="create")
+    assert accessible
+
+
+def test_user_group2_read_problematic_assignment(user_group2, problematic_assignment):
+    accessible = problematic_assignment.is_accessible_by(user_group2, mode="read")
+    assert accessible
+
+
+def test_user_group2_update_problematic_assignment(user_group2, problematic_assignment):
+    accessible = problematic_assignment.is_accessible_by(user_group2, mode="update")
+    assert accessible
+
+
+def test_user_group2_delete_problematic_assignment(user_group2, problematic_assignment):
+    accessible = problematic_assignment.is_accessible_by(user_group2, mode="delete")
+    assert accessible
+
+
+def test_super_admin_user_create_problematic_assignment(
+    super_admin_user, problematic_assignment
+):
+    accessible = problematic_assignment.is_accessible_by(
+        super_admin_user, mode="create"
+    )
+    assert accessible
+
+
+def test_super_admin_user_read_problematic_assignment(
+    super_admin_user, problematic_assignment
+):
+    accessible = problematic_assignment.is_accessible_by(super_admin_user, mode="read")
+    assert accessible
+
+
+def test_super_admin_user_update_problematic_assignment(
+    super_admin_user, problematic_assignment
+):
+    accessible = problematic_assignment.is_accessible_by(
+        super_admin_user, mode="update"
+    )
+    assert accessible
+
+
+def test_super_admin_user_delete_problematic_assignment(
+    super_admin_user, problematic_assignment
+):
+    accessible = problematic_assignment.is_accessible_by(
+        super_admin_user, mode="delete"
+    )
+    assert accessible
+
+
+def test_group_admin_user_create_problematic_assignment(
+    group_admin_user, problematic_assignment
+):
+    accessible = problematic_assignment.is_accessible_by(
+        group_admin_user, mode="create"
+    )
+    assert accessible
+
+
+def test_group_admin_user_read_problematic_assignment(
+    group_admin_user, problematic_assignment
+):
+    accessible = problematic_assignment.is_accessible_by(group_admin_user, mode="read")
+    assert accessible
+
+
+def test_group_admin_user_update_problematic_assignment(
+    group_admin_user, problematic_assignment
+):
+    accessible = problematic_assignment.is_accessible_by(
+        group_admin_user, mode="update"
+    )
+    assert accessible
+
+
+def test_group_admin_user_delete_problematic_assignment(
+    group_admin_user, problematic_assignment
+):
+    accessible = problematic_assignment.is_accessible_by(
+        group_admin_user, mode="delete"
+    )
+    assert accessible
+
+
+# These tests are commented out because they require email / twilio clients
+# to be set up, which is not done by default.
+"""
+def test_user_create_invitation(user, invitation):
```

Comment:
This has come up a couple of times I think, but it would be good to figure out a way to mock out third-party API calls at some point. Not in this PR, but in some future one, of course. Just wanted to note it here.